sentences/da/binary_sensor_HassGetState.yaml

```
@@ -0,0 +1,911 @@
+          domain: binary_sensor
+          device_class: cold
+          state: "on"
+
+      - sentences:
+          - "hvilke[n] [sensorer] er <koldt> [i <område>]"
+          - "hvad er <koldt> [i <område>]"
+        response: which
+        slots:
+          domain: binary_sensor
+          device_class: cold
+          state: "on"
+
+      - sentences:
+          - "<hvormange> [sensorer] er <koldt> [i <område>]"
+        response: how_many
+        slots:
+          domain: binary_sensor
+          device_class: cold
+          state: "on"
+
+      # Connectivity
+      - sentences:
+          - "er <navn> {bs_connectivity_states:state} [i <område>]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: connectivity
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      - sentences:
+          - "er [der] <nogen> <enheder> {bs_connectivity_states:state} [i <område>]"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      - sentences:
+          - "er alle <enheder>[ne] {bs_connectivity_states:state} [i <område>]"
+        response: all
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      - sentences:
+          - "hvilke[n] <enheder> er {bs_connectivity_states:state} [i <område>]"
+        response: which
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      - sentences:
+          - "<hvormange> <enheder> er {bs_connectivity_states:state} [i <område>]"
+        response: how_many
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      # Door
+      - sentences:
+          - "er <navn> {bs_door_states:state} [i <område>]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: door
+        slots:
+          domain: binary_sensor
+          device_class: door
+
+      # Garage door
+      - sentences:
+          - "er <navn> {bs_garage_door_states:state} [i <område>]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: garage_door
+        slots:
+          domain: binary_sensor
+          device_class: garage_door
+
+      # Gas
+      - sentences:
+          - "er <navn> {bs_gas_states:state} [i <område>]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: gas
+        slots:
+          domain: binary_sensor
+          device_class: gas
+
+      - sentences:
+          - "er [der] [nogen] gas sensorer {bs_gas_states:state} [i <område>]"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: gas
+
+      - sentences:
+          - "er [der] [noget] gas i <område>"
+          - "er [der] [noget] gas detekteret i <område>"
```

We allow `(detekteret|fundet|registreret)` for gas etc., so we should match them here (and other places) as well.

```suggestion
          - "er [der] [noget] gas (detekteret|fundet|registreret) i <område>"
```

Maybe a `<gas_fundet>` expansion rule could be useful, and a similar `<gas_ikke_fundet>` expansion rule for the negative cases.